### PR TITLE
MAT-7302 integrated backend with frontend for associating cms id

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE PR

Jira Ticket: [MAT-7302](https://jira.cms.gov/browse/MAT-7302)
(Optional) Related Tickets:

### Summary
integrated backend with frontend for associating cms id

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/measureLanding/measureList/associateCmsIdDialog/AssociateCmsIdDialog.tsx [hunk 1]
```diff
@@ -1,88 +1,121 @@
...
+                />
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </MadieDialog>
+
+      <AssociateCmsIdConfirmationDialog
+        open={associateCmsIdConfirmationDialogOpen}
+        onContinue={async () => {
```
Comment (RohitKandimalla) [inline src/components/measureLanding/measureList/associateCmsIdDialog/AssociateCmsIdDialog.tsx]: Why do we need a async function here ?

File: src/components/measureLanding/measureList/MeasureList.tsx [hunk 2]
```diff
@@ -816,6 +816,34 @@ export default function MeasureList(props: {
     setOpenAssociateCmsIdDialog(true);
   };
 
+  const handleCmsIdAssociationContinueDialog = (
```
Comment (RohitKandimalla) [inline src/components/measureLanding/measureList/MeasureList.tsx]: Not a big deal but the naming could be simplified

File: src/components/measureLanding/measureList/associateCmsIdDialog/AssociateCmsIdDialog.tsx (hunk 1 above)
Comment (sb-prateekkeerthi) [reply src/components/measureLanding/measureList/associateCmsIdDialog/AssociateCmsIdDialog.tsx]: I'll remove that, it is not necessary

File: src/components/measureLanding/measureList/MeasureList.tsx (hunk 2 above)
Comment (sb-prateekkeerthi) [reply src/components/measureLanding/measureList/MeasureList.tsx]: I'll change it to handleCmsIdAssociation

**Review by RohitKandimalla — APPROVED** [review]

**Pull request merged**